pkg/apis/storageos/v1/storageoscluster_types.go

```diff
@@ -202,6 +205,20 @@ type StorageOSClusterSpec struct {
 
 	// Disable StorageOS scheduler extender.
 	DisableScheduler bool `json:"disableScheduler,omitempty"`
+
+	// MinimumPidLimit is used to set the minimum PID limit required for
+	// StorageOS to start. The PID limit is set by the Kubernetes distribution
+	// or container runtime to limit the number of PIDs (including threads) that
+	// can be created by a single container.
+	//
+	// In most environments this will be at least the StorageOS recommended
+	// minimum of 32768, but the limit can be as low as 1024 in CRI-O based
+	// distributions such as Openshift.
+	//
+	// Before changing this value, consult the StorageOS documentation.  A
+	// negative value disables the check entirely.  Unset or '0' uses the
+	// StorageOS default.
+	MinimumPidLimit int `json:"minimumPidLimit,omitempty"`
```

Comment:
I'm not sure we want to add to the CRD until we set to enforcing as plans may change how to do it - thoughts?

Reply:
Even if the position is to have it disabled by default with -1, or using the defaults with 0, I have the impression that the CRD will need the flag anyway. And then it is the defaults in the operator that decide what to do. 
Since it has been agreed that PRE has to be able to disable it, having the CRD seems like the only option at the moment to expose that possibility, regardless of the implementation, and who executes the check/barrier.

Reply:
yes, we will almost certainly re-intoduce as-is, but for now there's no benefit to including it, so we agreed to remove.